spec/lib/task-data/tasks/create-raid-spec.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe(require('path').basename(__filename), function () {
+    var base = require('./base-tasks-spec');
+    //var hogan = require('hogan.js');
+
+    base.before(function (context) {
+        context.taskdefinition = helper.require('/lib/task-data/tasks/create-raid.js');
+    });
+
+    describe('task-data', function () {
+        base.examples();
+    });
+
+    describe('task options data', function () {
+        it('should have an options', function() {
+            expect(this.taskdefinition).to.have.property('options');
+            expect(this.taskdefinition.options).to.be.an('Object');
+        });
+
+        it('should have commands option', function() {
+            expect(this.taskdefinition.options).to.have.property('commands');
+            expect(this.taskdefinition.options.commands).to.be.an('Array').with.length(1);
+        });
+
```

Comment:
I'd like to see tests for the create/delete raid tasks actually define the tasks and render them, then compare their commands output against the expected, that way we know the rendering results in the command we want, and it also provides some informal documentation to future developers about what we expect the template to look like after being rendered. Also, we protect ourselves in case anyone makes changes to the command string in the future.